src/module.tsx

```diff
@@ -0,0 +1,23 @@
+import { lazy } from 'react';
+import { AppPlugin } from '@grafana/data';
+
+const App = lazy(async () => {
```

Comment:
Hm, I think the `lazy` breaks preloading, thus links from Explore and panels don't work with this PR.